pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -380,6 +380,8 @@ When you provision the cluster, the configured pod and service subnets will be a
 
 Confirm that your [Calico installation is correct][calico-install].
 
+**Note:** When you complete this procedure, move on to [Make Cluster Self-managed](https://archive-docs.d2iq.com/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/self-managed/) to continue the process.
```

Comment:
I don't know if we want the colored background and note format not, but just in case we do: 
```suggestion
<p class="message--note"><strong>NOTE: </strong>When you complete this procedure, move on to <a href="https://archive-docs.d2iq.com/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/self-managed/">Make Cluster Self-managed</a> to continue the process.</p>
```